backend/models/BlacklistToken.js

```diff
@@ -10,52 +10,52 @@ const __filename = fileURLToPath(import.meta.url);
 
 const blacklistCollection = getBlacklistCollection;
 
-const BlacklistToken = {
-  addTokenToList: async (token, expires) => {
-    try {
-      await blacklistCollection.insertOne({
-        token,
-        expires: Date.now(expires * 1000),
-      });
-    } catch (error) {
-      throw await errorHelpers.transformDatabaseError(
-        error,
-        __filename,
-        'addTokenToList',
-      );
-    }
-  },
+const blackListToken = {
```

Comment:
first lets fix spacing, then lets change the naming from blackList to blockList and others blacklistCollection to blockListCollection, please.